src/common/res/features/highlight-negatives-negative/main.js

```diff
@@ -1,38 +1,42 @@
 (function poll() {
   if ( typeof ynabToolKit !== "undefined" && ynabToolKit.pageReady === true ) {
+  
     ynabToolKit.highlightNegativesNegative = (function(){
+    
       return {
         invoke: function() {
-          var availableBalances = $('.budget-table-cell-available-div.user-data');
-          var categoryBalance = {};
-
-          $(availableBalances).each(function () {
-            categoryBalance = $(this).find('.user-data.currency').html(); // get the value
-              categoryBalance = Number( categoryBalance.replace(/[^\d.-]/g, '') ); // force data type as number
-
-
-            if ( categoryBalance < 0 ) {
-
-                if ( $(this).find('.user-data.currency').hasClass('cautious') ) {
-                  $(this).find('.user-data.currency').removeClass('cautious').addClass('negative');
-                }
-            }
-
-          });
+        
+          // look in budget, inspector
+      	  $.each($('.budget-table-cell-available-div.user-data .cautious, .budget-inspector .cautious'), function(key, val) {
+      		
+      		var num = $(this).text();
+      		if (parseInt(num.replace('$', '')) < 0 )
```

Comment:
Probably a better way to do this would be to use YNAB's built in unformat option so if someone's values look like 3,32€ it'll still work.

So this line would look like:

```javascript
if (ynab.unformat(num) < 0)
```